eng/testing/tests.browser.targets

```diff
@@ -27,6 +27,9 @@
     <InstallChromeForTests Condition="'$(InstallChromeForTests)' == '' and
                                         ('$(ContinuousIntegrationBuild)' != 'true' or Exists('/.dockerenv')) and
                                         '$(Scenario)' == 'WasmTestOnBrowser'">true</InstallChromeForTests>
+    <InstallV8ForTests Condition="'$(InstallV8ForTests)' == '' and
+                                        ('$(ContinuousIntegrationBuild)' != 'true' or Exists('/.dockerenv')) and
```

Comment:
this dockerenv check bothers me, but i guess it's fine since we were already doing it? it feels too magical

Reply:
That's the common way I found of detecting that you are running in a codespace.

Reply:
Maybe add a comment explaining that